apps/cnquery/cmd/builder/parse.go

```diff
@@ -389,6 +389,29 @@ func ParseTargetAsset(cmd *cobra.Command, args []string, providerType providers.
 	case providers.ProviderType_GCP:
 		connection.Backend = providerType
 
+		// env vars have precedence over the --credentials-path arg
+		credsPath := readEnvs("GOOGLE_APPLICATION_CREDENTIALS", "GOOGLE_CREDENTIALS")
```

Comment:
I added `GOOGLE_APPLICATION_CREDENTIALS` as per the [ADC](https://cloud.google.com/docs/authentication/application-default-credentials) docs